content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2396,6 +2396,18 @@ only onto these nodes, or to avoid running Pods on the control plane.
 If this label is set, the [EndpointSlice controller](/docs/concepts/services-networking/topology-aware-routing/#implementation-control-plane)
 ignores that node while calculating Topology Aware Hints.
 
+### kubernetes.io/role: control-plane
+
+Type: Label
+
+Example: `kubernetes.io/role: control-plane`
+
+Used on: Node
+
+A marker label similar to `node-role.kubernetes.io/control-plan` to indicate  node based on
+their work type of load etc. For example a worker node could be database, frontend etc.
```

Comment:
I don't think this explanation is correct.

Reply:
Please revise this explanation @kundan2707. Do you know how to liaise with SIG Architecture on this?

Reply:
@sftim i will get input from SIG-architcture slack channel and revise it based on that